README.md

```diff
@@ -92,6 +93,36 @@ The above output includes only software that is visible in the container (i.e.,
 syft <image> --scope all-layers
 ```
 
+#### Format conversion - __experimental__
+
+The ability to convert existing SBOMs means you can create SBOMs in different formats quickly, without the need to regenerate the SBOM from scratch, which may take significantly more time.
+
+``` bash
```

Comment:
nit: I just noticed the `bash` interpreter is making the synopsis syntax look strange since it's not real bash. Maybe we just use `text` or something here.